torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Callable, Union, Dict
+from torch.fx.graph_module import GraphModule
+from torch.fx.node import Node
+from torch.fx.graph import Graph
+
+Target = Union[Callable[..., Any], str]
+
+def replace_target_nodes_with(
+    fx_module: GraphModule,
+    old_op: str,
+    old_target: Target,
+    new_op: str,
+    new_target: Target,
+):
+    """Modifies all nodes in fx_module.graph.nodes which match the specified op code and target,
+    and updates them to match the new op code and target"""
+    new_graph = Graph()
+    val_map : Dict[Node, Node] = {}
+    for node in fx_module.graph.nodes:
+        if node.op == old_op and node.target == old_target:
+            new_args = tuple([val_map[arg] for arg in node.args])
```

Comment:
`args` and `kwargs` may be collection types composed of `Node`s that need to be switched over at any level of nesting. The way to process these is with `map_arg` https://github.com/pytorch/pytorch/blob/9e30a76697ddedb46887a92559b317840bce6804/torch/fx/graph.py#L55

So something like

```
args = torch.fx.graph.map_arg(node.args, lambda n: val_map[n])
kwargs = torch.fx.graph.map_arg(node.kwargs, lambda n: val_map[n])
val_map[node] = new_graph.create_node(new_op, new_target, args, kwargs, node.name)
```